src/Introspection/QueryRootFields.hack

```diff
@@ -0,0 +1,16 @@
+namespace Slack\GraphQL\Introspection;
+
+use namespace Slack\GraphQL;
+
+final abstract class QueryRootFields {
+    <<GraphQL\QueryRootField('__schema', 'Schema introspection')>>
+    final public static function getSchema(): ?__Schema {
+        // TODO: switch to taking $schema as first param?
+        return new \Slack\GraphQL\Test\Generated\Schema();
```

Comment:
Yeah this seems... problematic. I don't know how we can get around this without injecting the schema as an argument.

Reply:
It might be better to avoid using GraphQL\QueryRootField + codegen for these 2 fields. Instead, we could add a bit of extra codegen code to the generator for the Query root type, which would add these 2 fields to the generated output (so the generated output would be all the normal root fields parsed from various files + these 2 hardcoded fields).

Reply:
Yea, I'll leave this like this for this PR and change in a follow up since this is pretty straight forward

Reply:
Oooh, that makes a lot of sense.